analyses/cellranger-analysis/submit-multiple-jobs.sh

```diff
@@ -1,25 +1,23 @@
 #!/bin/bash
 
 #BSUB -P run_cellranger
-#BSUB -J submitter
+#BSUB -J submit-multiple-jobs
 #BSUB -q standard
 #BSUB -n 1
 #BSUB -R "rusage[mem=2500]"
-#BSUB -o submitter.out
-#BSUB -e submitter.err
-#BSUB -cwd "./sc-rna-seq-snap/analyses/cellranger-analysis"
+#BSUB -o submit-multiple-jobs.out
+#BSUB -e submit-multiple-jobs.err
+#BSUB -cwd "/research/dept/dnb/core_operations/Bioinformatics/achroni/GitHub/pipeline-testing/sc-rna-seq-snap/analyses/cellranger-analysis"
```

Comment:
Again, just a reminder to remove these paths when merging approved PR

Reply:
This has been addressed here https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap/commit/6c271f5863d698cbb52bd628dcf3842cf2313baf.